setup.py

```diff
@@ -30,7 +30,7 @@ def find_stub_files(name: str) -> List[str]:
 
 # Keep compatible-mypy major.minor version pinned to what we use in CI (requirements.txt)
 extras_require = {
-    "compatible-mypy": ["mypy==1.5.*"],
+    "compatible-mypy": ["mypy>=1.5,<1.7"],
```

Comment:
```suggestion
    "compatible-mypy": ["mypy==1.6.*"],
```

Reply:
Same as in https://github.com/typeddjango/django-stubs/pull/1764

```suggestion
    "compatible-mypy": ["mypy~=1.6.0"],
```